### ISSUE TITLE
Different clothing for different girls

### ISSUE THREAD

**Issue by FamTroder** [FamTroder/FamMel#102]

**Comment by M0nkey5ee** [comment]
Cool idea but sounds like a nightmare to code down the line (if different clothing should add different effects)

**Comment by Forget-About-Me2** [comment]
Yes and no, the current refactor I'm working on should at least allow for the descriptions to be more easily generalised down the line. But yeah I do think it would require some thinking and planning and I don't think this has a big priority
